.idea/GitAssemblyUtil.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do the example files need to be so huge? Is it possible to cut them down to much smaller files and still have the tests work?

Reply:
I cut down the assembly test data for the metagenome from MB to KB.